test/gateway/eip20_gateway/progress_stake_with_proof.js

```diff
@@ -123,6 +123,7 @@ contract('EIP20Gateway.progressStakeWithProof()', function (accounts) {
     let burner = NullAddress;
 
     bountyAmount = new BN(proofData.gateway.constructor.bounty);
+    penaltyAmount = bountyAmount.mul(new BN(150)).div(new BN(100));
```

Comment:
1. penaltyAmount = `bountyAmount.muln(1.5);`
2. we should declare `penaltyAmount` variable in line 40.